sdk/maps/maps-render-rest/generated/index.ts

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-import MapsRenderClient from "./mapsRenderClient";
```

Comment:
huh that's surprising to me - how come this file is deleted?

Reply:
Ah I think I get it. Is this a conflict between "all files must be under `src` and [customization v2 (internal)](https://microsoft.sharepoint.com/:fl:/s/d2e0cc81-28c8-4a82-99d9-ca0dd295e2db/EXZ9gfh_8htMlfbQd6eg4QsBUYsqnFHVBlmwd1Z7xDeGJg?e=hUFN9D&nav=cz0lMkZzaXRlcyUyRmQyZTBjYzgxLTI4YzgtNGE4Mi05OWQ5LWNhMGRkMjk1ZTJkYiZkPWIlMjF5UHE5d2tMRE5rT3ZGYXN1a0JTWTBPYkRyajNGbklCTmptdGJRZmxWS1VRbFB3LTFjd0x1U29EWWFSdk5MZ0lIJmY9MDEzUzdQV1EzV1BXQTdRNzdTRE5HSkw1V1FPNlQyQllJTCZjPSUyRiZhPUxvb3BBcHAmeD0lN0IlMjJ3JTIyJTNBJTIyVDBSVFVIeHRhV055YjNOdlpuUXVjMmhoY21Wd2IybHVkQzVqYjIxOFlpRmFTR015ZUdGdVgyZFZkVTF6ZEVSUFEwZHBNMG95TmtSRFQyZFdUa1ZHVEcxTU9EaFJRMnhsU25CcVNWaHFPSE5OYVVWSVZGbFVMVFp0TkVaWFltUTFmREF4UkVkR01rOUZORlJGVFZFME4xSkdRVGRhUlRKUFJrNURSRFJXV0RZME1rVSUzRCUyMiUyQyUyMmklMjIlM0ElMjIyOTE2YWViZS1iNzgzLTRlNmEtOTg1Yy1iNjczYTdiNWQxMGElMjIlN0Q%3D) ? If so, we'll have to fix one or the other if we can't have files _not_ under src cc @timovv

Reply:
Already discussed and there is no customization required for these libraries, hence why we can move directly into src/generated.